.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Bootstrap app, run linters and tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node and restore cached dependencies
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

Comment:
Should we use the [build matrix](https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs) to test the template working on both LTS and the latest node version? Or do you want to support only the LTS version?

Reply:
Updated in [8447d68](https://github.com/nimblehq/react-templates/pull/72/commits/8447d6891dfa2ed0a2745fa6e85631c8e899cb9a) to also include node 14 🙌🏼